modules/nf-core/openms/idripper/tests/main.nf.test

```diff
@@ -0,0 +1,86 @@
+nextflow_process {
+
+    name "Test Process OPENMS_IDRIPPER"
+    script "../main.nf"
+    process "OPENMS_IDRIPPER"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "openms"
+    tag "openms/idmerger"
+    tag "openms/idripper"
+
+    test("proteomics - split - idxml") {
+        setup {
```

Comment:
if a setup block is to be used for multiple tests....you can move it beyond any test scope and place it above so it becomes global....read more about global setup [here](https://www.nf-test.com/docs/testcases/setup/)

Reply:
Thanks for the hint! Makes way more sense